src/dev/build/tasks/clean_tasks.js

```diff
@@ -55,6 +56,21 @@ export const CleanTypescriptTask = {
   },
 };
 
+export const CleanNodeTask = {
+  description:
+    'Cleaning npm from node',
+
+  async run(config, log, build) {
+    for (const platform of config.getPlatforms()) {
+      await deleteAll(log, [
+        build.resolvePathForPlatform(platform, 'node/lib/node_modules/npm'),
```

Comment:
Maybe here we can just delete the entire `node_modules` folder. In the distributable I believe we don't want the user to use any node_module ever shipped with node anyway 😄

Reply:
Pushed in https://github.com/elastic/kibana/pull/21746/commits/adfb2d53c8914cc75ba5e531fb1c1848e6b34f73